docker_daemon/CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # CHANGELOG - docker_daemon
 
-1.2.0 / Unreleased
+1.3.0 / Unreleased
+==================
+### Changes
+
+* [FEATURE] collect kube_container_name by default in docker_daemon check, like kubernetes does. See [#553][]
+* [FEATURE] add kube_container_name tag to kubernetes and docker integrations. See [#509][], thanks [@sophaskins][]
+* [IMPROVEMENT] remove NomadUtil & ECSUtil from docker_daemon, MetadataCollector proxies them. See [#486][]
+* [IMPROVEMENT] use the new orchestrator.Tagger class to retrieve the Mesos tags for docker metrics. See [#466][]
```

Comment:
I'd rephrase 

[IMPROVEMENT] Mesos: extract marathon_app, chronos_job and chronos_job_owner tags.. See [#466][]